sdk/storage/azure-storage-file-datalake/pom.xml

```diff
@@ -76,13 +76,13 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-test</artifactId>
-      <version>1.0.0</version>  <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
+      <version>1.1.0-beta.1</version>  <!-- {x-version-update;com.azure:azure-core-test;dependency} -->
```

Comment:
I'm assuming this means we are shipping this version of Core to get Data Lake out?

Reply:
@mitchdenny , Datalake had to revert this version temporarily like 2 hours go in order to ship. I'm just changing it back to what it was previously since they shipped. The updating post GA snafu'd their original release. This is the value that it was prior to them swapping so they could build and target the released version.

Reply:
@mitchdenny the changed things back to build and release, that was done in this [PR](https://github.com/Azure/azure-sdk-for-java/pull/6210) . I'm putting the versions back to the next version.